### PR TITLE
feat: support tf 2.6 in TFKerasTrial, update env images.

### PR DESCRIPTION
**Pull request by ioga**
## Description

- In 2.6, import keras internals from `keras` instead of `tf.python.keras`.
- Update tf env images with security upgrades: 2.4.3, 2.5.1.
- Add tf env image for 2.6.

## Test Plan

- circleCI test for 2.6 series.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: harness/determined/keras/_tf_keras_trial.py [hunk 1]
```diff
@@ -13,19 +13,30 @@
...
-)
-from tensorflow.python.keras.utils.mode_keys import ModeKeys
 
 import determined as det
 from determined import horovod, keras, util, workload
 from determined._tf_rng import get_rng_state, set_rng_state
 from determined.common import check
 from determined.horovod import hvd
 
+# In TF 2.6, we have to import some keras internals directly from `keras`.
```
Comment (rb-determined-ai) [inline harness/determined/keras/_tf_keras_trial.py:23]: sigh.

**Review by rb-determined-ai — APPROVED** [review]

**Comment by ioga** [comment]
nightlies are still running but the rest is green: https://app.circleci.com/pipelines/github/determined-ai/determined/14750/workflows/52d1d9d1-a99b-462c-82d0-83e9c7b57a19

**Review by rb-determined-ai — APPROVED** [review]
still approved.
